OSCreceiver.h

```diff
@@ -33,10 +33,10 @@ class OSCreceiver :
 	
 	bool start();
 	void stop();
-	
-	bool hasWaitingMessages();
-	bool getNextMessage(gdOscMessage*);
-	
+		
+	bool has_waiting_messages();
```

Comment:
Let's stick to camelCase?

Reply:
I was using the "_" separated to follow gdscript way. If you prefer camel case, no problem :)

Reply:
oh, yeah, you're right!